src/main/test/java/com/playtika/homework6/ElMarketTest.java

```diff
@@ -0,0 +1,56 @@
+package com.playtika.homework6;
```

Comment:
Why you've added to `test` package? Move to `java`

Reply:
fixed